vault/cluster.go

```diff
@@ -363,7 +375,7 @@ func (c *Core) ClusterTLSConfig(ctx context.Context, repClusters *ReplicatedClus
 	}
 
 	clientLookup := func(requestInfo *tls.CertificateRequestInfo) (*tls.Certificate, error) {
-		//c.logger.Trace("core: performing client cert lookup")
+		//c.logger.Trace("performing client cert lookup")
```

Comment:
Remove or enable commented out code.

Reply:
@jefferai ditto question for this and the one below.

Reply:
Going to go ahead and strip it - just let me know if I should revert that.